core/codegen/module.go

```diff
@@ -98,6 +103,13 @@ func NewModule(name string, target *device.ABI) *Module {
 	m.Types.emptyStructField = Field{"empty-struct", m.Types.Int8}
 	m.Types.m = m
 
+	if true {
```

Comment:
if true { ? 

Can we get a comment as to why?

Reply:
Good spot - bad merge!

Reply:
Fixed.